server/app/services/geo/esri/EsriServiceAreaState.java

```diff
@@ -0,0 +1,11 @@
+package services.geo.esri;
+
+/** Defines the states of inclusion for an address in a service area. */
+public enum EsriServiceAreaState {
+  IN_AREA,
+  NOT_IN_AREA,
+  // FAILED indicates that the service area validation check failed.
+  // This can happen if the service area valiation option config does not load properly,
+  // or if the external Esri service returns a non 200 status
+  FAILED
+}
```

Comment:
**suggestion:** since the serialization format is critical to the feature working correctly let's make it explicit. Also a nit is that service area validation is not necessarily coupled to ESRI if we integrate with some other GIS system.

```suggestion
/** The result of checking an address for membership in a service area. */
public enum ServiceAreaState {
  // The address is in the service area.
  IN_AREA("InArea"),
  // The address is not in the service area.
  NOT_IN_AREA("NotInArea"),
  // The check failed for technical reasons.
  FAILED("Failed");

  private final String serializationFormat;

  ServiceAreaState(String serializationFormat) {
    this.serializationFormat = checkNotNull(serializationFormat);
  }

  /** The string representation of this state for storage in the database. */
  public String getSerializationFormat() {
    return this.serializationFormat;
  }
}
```

Reply:
Good call. Moved this and ServiceAreaInclusion up one level out of /esri.